Sources/SnapshotTestCase.swift

```diff
@@ -211,6 +241,17 @@ private extension String {
     }
 }
 
+private extension CGRect {
```

Comment:
Let's move extensions to separate files. Group these extensions by folders with framework names. (`Metal/MTLBlaBla+Extensions.swift`, `CoreGraphics/CGrect+Extensions.swift`),